The_Python_Communist_Revolution_Notebook.ipynb

```diff
@@ -0,0 +1,437 @@
+        "    uncompressed_string = str(base64.decodebytes(zlib.decompress(compressed_data)), encoding='ascii')\n",
+        "    return uncompressed_string\n",
```

Comment:
WTH is this!?
It looks like imperialist spy, or sabotage!

Reply:
It's just some song lyrics, print it out and you'll see.
Don't worry, there isn't any know python exploits that use simple strings.

Reply:
I have checked, they are effectively lyrics.

Reply:
I'm overriding the __str__ function of the class to be able to perform print(OrdinaryPeople()) and each class returns something different.
The class WorkingClassHeroes() when printed, returns John Lennon's song Working Class Hero but I compressed the string to make it like an easter egg.
You can check in the notebook the section that invites you to guess what the WorkingClassHeroes class prints and to uncomment the line to print the song.